home0.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import ("fmt")
+
+var a, x, y, p, sq, x1, y1 int
+
+func main() {
+	// you can change this values
+	a = 5
+	x = 0
+	y = 0
+	// now we start our calculations
+	sq = a*a

```

Comment:
It should be done via struct + methods
Feel free to contact mentors in slack if required.